tests/tests_geomstats/test_visualization_beta_distributions.py

```diff
@@ -0,0 +1,90 @@
+"""Unit tests for visualization."""
+import os
+import sys
+
+import matplotlib
+import matplotlib.pyplot as plt
+
+import geomstats.backend as gs
+import tests.conftest
+from geomstats.information_geometry.beta import BetaDistributions
+from geomstats.visualization.beta_distributions import Beta
+
+sys.path.append(os.path.dirname(os.getcwd()))
+
+matplotlib.use("Agg")  # NOQA
+
+
+class TestVisualizationBeta(tests.conftest.TestCase):
+    def setup_method(self):
+        self.n_samples = 10
+        self.Beta = BetaDistributions()
+        self.beta_viz = Beta()
+
+    def test_plot_beta(self):
+        points = gs.random.rand(2, 2)
+        self.beta_viz.plot(points)
+
+    def test_scatter_beta(self):
+        num_points = gs.random.randint(2, 50)
```

Comment:
Use n_points instead of num_points, for consistency with geomstats' API (on this line and everywhere else)